static/src/javascripts/projects/common/modules/save-for-later.js

```diff
@@ -51,6 +51,15 @@ define([
         this.isContent = !/Network Front|Section/.test(config.page.contentType);
         this.userData = null;
         this.savedArticlesUrl = config.page.idUrl + '/saved-for-later-page';
+
+        _.bindAll(this,
+            'save',
+            'delete',
+            'onSaveArticle',
+            'onDeleteArticle',
+            'createSaveFaciaItemHandler',
+            'createDeleteFaciaItemHandler'
+        );
```

Comment:
I think the fact we are passing methods (not simply functions) around as callbacks and thereby losing implicit binding is a bit of a code smell. This could just be a file of functions without a `SaveForLater` object, which would be simpler because of no prototype nor binding. Not endorsing a bigger refactor, just pointing it out!